src/js/themes/hpe.js

```diff
@@ -950,7 +950,11 @@ export const hpe = deepFreeze({
       down: FormDown,
       up: FormUp,
     },
-    options: undefined,
+    options: {
```

Comment:
I think we want to leave this as undefined because we have "option" button kind which is used for this kind of button and applies some other styling as well. Thinking we should adjust the label weight in that object instead.

Reply:
I believe [this line](https://github.com/grommet/grommet-theme-hpe/pull/237/files#diff-0ea9b1217744b978fa7c1ac3ab44c113973e55fbe073fc22011857993680bb57R204) should change to 500.

Reply:
good insight I was not aware we had that button kind I will update.